<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>587</width>
    <height>687</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="txt_dni">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_apellidos">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_dni">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DNI/CIF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_nombre">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_nombre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_apellidos">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_apellidos_2">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>120</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dirección</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_direccion">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_apellidos_3">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>150</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Población</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_poblacion">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_apellidos_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_cp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_apellidos_5">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>210</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Provincia</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_provincia">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Murcia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Madrid</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_dni_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>270</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Teléfono</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_nombre_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Móvil</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellidos_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>E-Mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_telefono">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellidos_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WEB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_movil">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_web">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_hombre">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hombre</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_mujer">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mujer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_otros">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Otros</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_deudor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deudor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_credito">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Credito</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_observaciones">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtp_fnac">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellidos_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>F. Nac</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellidos_9">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>520</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Observaciones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aceptar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_limpiar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txt_dni</tabstop>
  <tabstop>txt_nombre</tabstop>
  <tabstop>txt_apellidos</tabstop>
  <tabstop>txt_direccion</tabstop>
  <tabstop>txt_poblacion</tabstop>
  <tabstop>txt_cp</tabstop>
  <tabstop>cmb_provincia</tabstop>
  <tabstop>txt_telefono</tabstop>
  <tabstop>txt_movil</tabstop>
  <tabstop>txt_email</tabstop>
  <tabstop>txt_web</tabstop>
  <tabstop>dtp_fnac</tabstop>
  <tabstop>rdb_hombre</tabstop>
  <tabstop>rdb_mujer</tabstop>
  <tabstop>rdb_otros</tabstop>
  <tabstop>chb_deudor</tabstop>
  <tabstop>chb_credito</tabstop>
  <tabstop>txt_observaciones</tabstop>
  <tabstop>btn_aceptar</tabstop>
  <tabstop>btn_limpiar</tabstop>
  <tabstop>btn_eliminar</tabstop>
  <tabstop>btn_salir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
